app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def decorator(*args, **kwargs) -> int | list:
+        el = str(args)
```

Comment:
Converting `args` to a string may not uniquely identify the arguments if they contain mutable objects or if the function accepts keyword arguments. Consider using a more robust method to create a unique key, such as hashing the arguments.